### DISCUSSION TITLE
Error with Raspbian install 1.1.9 on Bullseye

### DISCUSSION THREAD

**Discussion by tcoombs** [rustdesk/rustdesk#852] [Q&A]
I have two Raspbian systems. The first one installed without any problem but with the second system I get the an error when I install with sudo apt install -fy ./rustdesk-1.1.9-raspberry-armhf.deb.  At the end of the install script I see the following error:

W: Repository is broken: rustdesk:armhf (= 1.1.9) has no Size information

if I run rustdesk from terminal I see the following error:

rustdesk: error while loading shared libraries: libpulse-simple.so.0: cannot open shared object file: No such file or directory

I've tried the following to see if there are any broken dependancies but this has not found anything:
sudo dpkg --configure -a
sudo apt-get install -fy

I'm using bullseye version of Raspbian.

Any suggestions on how to fix this (I've already gone through the un-install re-install route...)

**Comment by rustdesk** [comment]
@Heap-Hop

**Comment by tcoombs** [comment]
Guys, any news on this problem, please?

**Comment by rustdesk** [comment]
Sorry, you have to figure out it yourself, we have no resource to figure out all platform problems, especially there are countless Linux distros and versions.

**Comment by rustdesk** [comment]
or build yourself: https://github.com/rustdesk/rustdesk/discussions/697#discussioncomment-2879261

**Comment by Heap-Hop** [comment]
It works fine on my Pi4, I just reset my Pi, 
```sudo apt install -fy ./rustdesk-1.1.9-raspberry-armhf.deb``` 
and run it on a new Raspbian OS(32bit)
didn't find any problem.

```
$ cat /etc/os-release 

PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

According to your infomation, maybe you need to check `libpulse-simple.so.0` like [this](https://www.google.com/search?q=libpulse-simple.so.0%3A+cannot+open+shared+object+file%3A+No+such+file+or+directory&sourceid=chrome&ie=UTF-8)

**Reply by tcoombs** [reply]
when you type "uname -m" on the terminal what do you see?

**Comment by tcoombs** [comment]
I think I've identified the problem. Rustdesk works fine on a 32bit version of Raspberry PI OS, but if I try to install on the 64 bit version of the OS I get the problem. I've now verified this on two separate installations of RaspberryPI's running 64 bit versions of the OS.

Therefore, only 32 bit seems to work for me.

**Reply by Heap-Hop** [reply]
> and run it on a new Raspbian OS(32bit)

Yes, only support 32-bit currently. https://github.com/rustdesk/rustdesk/issues/853#issuecomment-1164535620
